app/views/shared/_cart_list.html.erb

```diff
@@ -0,0 +1,27 @@
+<%#-- "carts" is the critical local variable for using this partial --%>
+<%#-- "role" (either 'requester' or 'approver') modifies the list slightly --%>
+<%#-- "closed" (bool) determines whether we're showing closed carts or not. --%>
+<%#-- "limit" is the maximum noumber of carts to show. --%>
+<table class="cart-list">
+  <tr class="header">
+    <th class="first" scope="col"><h5>Request</h5></th>
+    <th class="sixth" scope="col"><h5>Total Price</h5></th>
+    <th class="sixth" scope="col"><h5><%= if role=='requester' then 'To' else 'From' end%> </h5></th>
+    <th class="sixth" scope="col"><h5>Date</h5></th>
+  </tr>
+  <%- carts.each_with_index do |cart, index| %>
+    <%- if defined?(limit) && index >= limit then break end %>
```

Comment:
Can we use [`limit`](http://guides.rubyonrails.org/active_record_querying.html#limit-and-offset) on the query instead?